osu.Desktop/Program.cs

```diff
@@ -29,7 +30,19 @@ public static void Main(string[] args)
         {
             // run Squirrel first, as the app may exit after these run
             if (OperatingSystem.IsWindows())
+            {
+                var windowsVersion = Environment.OSVersion.Version;
+
+                if (windowsVersion.Major < 6 || (windowsVersion.Major == 6 && windowsVersion.Minor <= 1))
```

Comment:
Realm only supports Windows 8.1+ (https://www.mongodb.com/docs/realm/sdk/dotnet/#supported-platforms), so might as well change this to:
```suggestion
                if (windowsVersion.Major < 6 || (windowsVersion.Major == 6 && windowsVersion.Minor <= 2))
```

Reply:
Had a feeling that was the case, but couldn't readily find that info.